public/app/features/dashboard-scene/serialization/transformSaveModelToScene.ts

```diff
@@ -437,14 +446,38 @@ export function buildGridItemForPanel(panel: PanelModel): SceneGridItemLike {
     });
   }
 
+  const body = new VizPanel(vizPanelState);
+
   return new SceneGridItem({
     key: `grid-item-${panel.id}`,
     x: panel.gridPos.x,
     y: panel.gridPos.y,
     width: panel.gridPos.w,
     height: panel.gridPos.h,
-    body: new VizPanel(vizPanelState),
+    body,
   });
 }
 
 const isAdhocVariable = (v: VariableModel): v is AdHocVariableModel => v.type === 'adhoc';
+
+const debouncedDescriptionReporting = debounce(() => {
+  DashboardInteractions.panelDescriptionShown();
+}, 100);
```

Comment:
@axelavargas  - this is how I solved the description being rendered multiple times :)

Reply:
@ivanortegaalba it's a good idea. 

However, I think it's weird we report description-shown only once currently. Is this desired behavior, or is it because we hit some limitations? @axelavargas @thanos-karachalios ?

Reply:
I suggested that because [the current behavior is to report it once](https://github.com/grafana/grafana/pull/79078/files#diff-2e4d7a90c2dcc990dd68e1cc1e882ff084d676c94bc5f70f4a78424a2bd25bb9R44-R48). If now report it multiple times, it would suggest that people are using more the description than before, which is not true.

However, as soon as we agree this is the desired behavior, we can replace the previous with the current behavior, and that should be enough.

IMO, your approach works better. We decided to report it only once because we only wanted to know if the user found it.

Reply:
@axelavargas / @ivanortegaalba - brought back the current behavior parity with [a450d8a](https://github.com/grafana/grafana/pull/79078/commits/a450d8ac3f5975387b7be71db6f585846158a656)